slave.mk

```diff
@@ -761,7 +761,7 @@ $(addprefix $(TARGET_PATH)/, $(DOCKER_IMAGES)) : $(TARGET_PATH)/%.gz : .platform
 		PACKAGE_URL_PREFIX=$(PACKAGE_URL_PREFIX) \
 		SONIC_ENFORCE_VERSIONS=$(SONIC_ENFORCE_VERSIONS) \
 		TRUSTED_GPG_URLS=$(TRUSTED_GPG_URLS) \
-		scripts/prepare_docker_buildinfo.sh $* $($*.gz_PATH)/Dockerfile $(CONFIGURED_ARCH)
+		SONIC_VERSION_CONTROL_COMPONENTS=$(SONIC_VERSION_CONTROL_COMPONENTS) scripts/prepare_docker_buildinfo.sh $* $($*.gz_PATH)/Dockerfile $(CONFIGURED_ARCH)
```

Comment:
Not necessary to pass the environment value, the value can be retrieve from "src/sonic-build-hooks/buildinfo/config/buildinfo.config"
$ cat src/sonic-build-hooks/buildinfo/config/buildinfo.config
PACKAGE_URL_PREFIX=https://packages.trafficmanager.net/public/packages
SONIC_VERSION_CONTROL_COMPONENTS=deb,py2,py3,web